server/src/test/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsResponseTests.java

```diff
@@ -119,7 +119,7 @@ private GetSnapshotsResponse createTestInstance() {
             failures.put(repository, new ElasticsearchException(randomAlphaOfLength(10)));
         }
 
-        return new GetSnapshotsResponse(responses, failures);
+        return new GetSnapshotsResponse(responses, failures, null);
```

Comment:
I think we sometimes need to add a random `next` value here? Also, `assertEqualInstances` should be updated.